src/index.js

```diff
@@ -71,7 +71,7 @@ module.exports = (
   // we need to catch here because the plugin will
   // error if there's no tsconfig in the working directory
   try {
-    resolvePlugins.push(new TsconfigPathsPlugin({ silent: true }));
+    resolvePlugins.push(new TsconfigPathsPlugin({ silent: true, extensions: [".ts", ".tsx", ".js"] }));
```

Comment:
```suggestion
    resolvePlugins.push(new TsconfigPathsPlugin({ silent: true, extensions: SUPPORTED_EXTENSIONS }));
```

The docs mention that these should be the same.

Perhaps this should look for `allowJs` option before assigning the extensions.

> If you're using allowJs in tsconfig.json, or allow other non-TS extensions in webpack, make sure you set extensions option in sync with your webpack config.

Reply:
@styfle I believe I addressed both your suggestions, I did it in two steps, let me know if you want to keep the check against the tsconfig.json